psi4/driver/procrouting/sapt/fisapt_proc.py

```diff
@@ -140,6 +140,14 @@ def fisapt_fdrop(self):
     with open(geomfile, 'w') as fh:
         fh.write(xyz)
 
+    # write external potential geometries
+    if external_potentials is not None:
+        for frag in ["A", "B", "C"]:
+            if external_potentials.get(frag, None) is not None:
```

Comment:
This is minor, but I'd recommend saving `external_potentials.get(frag, None)` to a local variable for use in the below line.

Reply:
Fixed.